main.go

```diff
@@ -35,7 +35,6 @@ func main() {
 	}
 	utils.Version = version
 	fmt.Println(logo, " ")
-	fmt.Printf("version: %v\n\n", version)
```

Comment:
For time being I think we should Let it be. When ever we get the error logs from user if it contains the version it will be useful for debugging.

Reply:
Alright

Reply:
This is not yet reverted. I think I was not clear last time, we need to specify version every time we run any command in keploy for debugging purposes.

Reply:
Apologies for the oversight, Refactored